src/main/java/info/matai/models/configuration/Server.java

```diff
@@ -1,15 +1,49 @@
 package info.matai.models.configuration;
 
+import info.matai.models.enums.ServerTypes;
 import java.util.List;
 
 public class Server {
   private String mcVersion;
-  private String serverType;
+  private ServerTypes serverType;
+  private String serverBuild;
   private List<Plugin> plugins;
 
   public Server(String mcVersion, String serverType, List<Plugin> plugins) {
```

Comment:
fyi: Vu que c'est dans le constructeur, tu peux y aller directement avec sa valeur `this.mcVersion = mcVersion`